### PR TITLE
Callout campaign css change

### PR DESCRIPTION
**Pull request by akemitakagi**
## What does this change?
Added css changes to the campaign forms.

## Screenshots
**Closed**
picture 112
**Opened**
picture 113
**News**
picture 114
**Sport**
picture 115
**Opinion**
picture 116
**Culture**
picture 117
**Life and style**
picture 118

### Does this affect other platforms?
No

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?
No

### Accessibility test checklist

Colour contrast passed

### Tested
Local

### CODE REVIEW

File: static/src/javascripts/projects/journalism/views/campaignForm.html [hunk 1]
```diff
@@ -40,14 +40,13 @@ <h4 class="campaign--snippet__headline"> <%= data.title %></h4>
                 <label for="<%=field.name%>"
                        class="form_field_label">
                     <%= field.label %>
-                    <span><%= field.description %></span>
+                    <% if(field.description) { %>
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/journalism/views/campaignForm.html]: 👍

File: static/src/javascripts/projects/journalism/views/campaignForm.html [hunk 2]
```diff
@@ -40,14 +40,13 @@ <h4 class="campaign--snippet__headline"> <%= data.title %></h4>
...
+                        <span>(<%= field.description %>)</span>
+                    <% } %>
                 </label>
 
                 <% if(field.type === "textarea") { %>
-                <textarea name="field_<%=field.id%>"
-                <% if (field.required == "1") { %> required <% } %>
-                />
-                </textarea>
+                <textarea name="field_<%=field.id%>"<% if (field.required == "1") { %> required <% } %>/></textarea>
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/journalism/views/campaignForm.html]: Well done removing the extra whitespace 👍

File: static/src/stylesheets/module/journalism/_campaigns.scss [hunk 3]
```diff
@@ -1,25 +1,204 @@
...
+    p {
+        font-size: 14px;
+        line-height: 18px;
+        font-family: $f-serif-headline;
+        font-weight: 400;
+        padding-top: 2px;
+    }
+}
+
+article {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/journalism/_campaigns.scss]: You can drop that, what's inside is just the defaults setting, so:
```css
.speech-bubble { ... }
.content--pillar-news .speech-bubble { ... overrides ... }
```

File: static/src/stylesheets/module/journalism/_campaigns.scss [hunk 4]
```diff
@@ -1,25 +1,204 @@
...
+        }
+    }
+    .form_submit {
+        button {
+            color: $garnett-neutral-5;
+            background-color: $news-garnett-main-1;
+        }
+    }
+    .campaign--snippet__handle {
+        color: $garnett-neutral-5;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/journalism/_campaigns.scss]: Lines 76-77 can go as they are the same as the default setting

File: static/src/stylesheets/module/journalism/_campaigns.scss [hunk 5]
```diff
@@ -1,25 +1,204 @@
...
+        }
+    }
+    .form_submit {
+        button {
+            color: $garnett-neutral-5;
+            background-color: $sport-garnett-feature;
+        }
+    }
+    .campaign--snippet__handle {
+        color: $garnett-neutral-5;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/journalism/_campaigns.scss]: ditto

File: static/src/stylesheets/module/journalism/_campaigns.scss [hunk 6]
```diff
@@ -1,25 +1,204 @@
...
+article {
+    .speech-bubble {
+        color: $garnett-neutral-1;
+        background-color: $news-garnett-highlight;
+        &::after {
+            color: $garnett-neutral-1;
+            background-color: $news-garnett-highlight;
+        }
+    }
+    .form_submit {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/journalism/_campaigns.scss]: I'd be more comfortable if this was limited to a callout, e.g. `.campaign--snippet .form_submit button`

File: static/src/stylesheets/module/journalism/_campaigns.scss (hunk 3 above)
Comment (regiskuckaertz) [reply static/src/stylesheets/module/journalism/_campaigns.scss]: I think it's easier to follow if you specialise per item, so:
```css
.speech-bubble {
  ...
  .content--pillar-news & { ... }
  .content--pillar-news & { ... }
  .content--pillar-news & { ... }
  .content--pillar-news & { ... }
}
.campaign .form_submit button {
   ...
  .content--pillar-news & { ... }
  .content--pillar-news & { ... }
  .content--pillar-news & { ... }
  .content--pillar-news & { ... }
}
```

File: static/src/stylesheets/module/journalism/_campaigns.scss [hunk 7]
```diff
@@ -216,15 +404,22 @@
         text-align: right;
         display: flex;
         font-family: $f-sans-serif-text;
+        font-size: 12px;
+
         a {
             flex: auto;
-            align-self: center;
             color: #121212;
+            text-decoration: none;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/journalism/_campaigns.scss]: don't we inherit the styles from frontend? (i.e. the underline)

**Review by regiskuckaertz — CHANGES_REQUESTED** [review]
a few minor things

File: static/src/javascripts/projects/journalism/views/campaignForm.html (hunk 1 above)
Comment (akemitakagi) [reply static/src/javascripts/projects/journalism/views/campaignForm.html]: @regiskuckaertz Please review the change.

File: static/src/javascripts/projects/journalism/views/campaignForm.html (hunk 2 above)
Comment (akemitakagi) [reply static/src/javascripts/projects/journalism/views/campaignForm.html]: @regiskuckaertz Please review the change.

**Review by regiskuckaertz — APPROVED** [review]
Well done @akemitakagi

**Pull request merged**